### PR TITLE
tests: rework main test and add cases

### PR DESCRIPTION
**Pull request by nickkbell**

### CODE REVIEW

File: tests/objects/MainPage.js [hunk 1]
```diff
@@ -20,7 +55,12 @@ export class MainPage {
         // missile click for closing select list
         codecSelect
             ? await this.page.getByRole('option', {name: 'analog'}).click()
-            : await this.page.locator('#root').click();
+            : await this.page.click('body', {
+                position: {
```
Comment (DarkPark) [inline tests/objects/MainPage.js:59]:
```suggestion
                position: {x: 100, y: 100}
```

seems good as well

File: tests/specs/main/commands.spec.js [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+    test.describe('check mtx frame view', () => {
+        test.beforeEach(async ( {page, baseURL} ) => {
+            await page.goto(baseURL);
+            await mainPage.selectMtxCodec();
+        });
+
+        test('validate commands for mtx codec', async ( {page} ) => {
+            const $commandsOptions = await mainPage.getAllSelectOption(MainPage.command);
+
+            await await expect(page.locator('[name="accessKey"]')).toHaveValue('00 01 02 03 04 05 06 07 08 09 0a 0b 0c 0d 0e 0f');
```
Comment (DarkPark) [inline tests/specs/main/commands.spec.js:72]: constant for aes key?

File: tests/specs/main/logs.spec.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
+import {expect, test} from '@playwright/test';
+import {MainPage} from '../../objects/MainPage.js';
+
+
+const link = `jooby-gui#logs=NobwRAxg9gtjCGA7AJgFQJ4AcCmYBcYS8ANlAOZgA0YAFvAE7IDuD2GO+AjAEzXICW9bBAAu
```
Comment (DarkPark) [inline tests/specs/main/logs.spec.js:5]:
```js
const something = `
    some
    long
    string
`.replace()
```

File: tests/specs/main/commands.spec.js [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+        });
+
+        test('add commands to frame and clear', async ( {page} ) => {
+            await mainPage.createMessage('getEventStatus', 'downlink');
+
+            const $accessLevels = await mainPage.getAllSelectOption('Access level');
+
+            // missile click
+            await page.locator('#menu-accessLevel div').first().click();
+            await await expect(mainPage.getSourceAddress()).toHaveValue('ff fe');
```
Comment (DarkPark) [inline tests/specs/main/commands.spec.js:85]: more await-s - more fun? )

**Review by aleks-and — DISMISSED** [review]

File: tests/objects/MainPage.js (hunk 1 above)
Comment (nickkbell) [reply tests/objects/MainPage.js]: ++

File: tests/specs/main/commands.spec.js (hunk 2 above)
Comment (nickkbell) [reply tests/specs/main/commands.spec.js]: ++

File: tests/specs/main/logs.spec.js (hunk 3 above)
Comment (nickkbell) [reply tests/specs/main/logs.spec.js]: ++

File: tests/specs/main/commands.spec.js (hunk 4 above)
Comment (nickkbell) [reply tests/specs/main/commands.spec.js]: yep
fixed it

**Review by DarkPark — APPROVED** [review]

**Review by DeadByElpy — APPROVED** [review]

**Pull request merged**